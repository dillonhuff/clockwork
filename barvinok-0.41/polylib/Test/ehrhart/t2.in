5 5
1 2 0 -1 0
1 -1 1 0 20
1 -1 -1 0 20
1 1 1 0 0
1 1 -1 0 0

1 3
1 1 0          # P>=0

P
2nd simple test

x >= n/2
x <= y <= -x
20-x <= y <= x-20
---------------------------------------
Domain :
         - P + 20 >= 0
         P  >= 0
 
Vertices :
[ 10,  10 ]
[ 10,  -10 ]
[ 20,  0 ]
[ P/2,  P/2 ]
[ P/2,  -P/2 ]
---------------------------------------
Domain :
         - P + 40 >= 0
         P  -20 >= 0
 
Vertices :
[ 20,  0 ]
[ P/2,  -P/2+20 ]
[ P/2,  P/2-20 ]

