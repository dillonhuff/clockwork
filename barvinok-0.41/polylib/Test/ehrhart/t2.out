---------------------------------------
Domain:
         - P + 20 >= 0
         P  >= 0

Vertices:
[ 10,  10 ]
[ 10,  -10 ]
[ 20,  0 ]
[ P/2,  P/2 ]
[ P/2,  -P/2 ]

Ehrhart Polynomial:
( -1/4 * P^2 + [ 0, -1/2 ]_P * P + [ 221, 883/4 ]_P )

---------------------------------------
Domain:
         P  -20 >= 0
         - P + 40 >= 0

Vertices:
[ 20,  0 ]
[ P/2,  -P/2+20 ]
[ P/2,  P/2-20 ]

Ehrhart Polynomial:
( 1/4 * P^2 + [ -21, -41/2 ]_P * P + [ 441, 1681/4 ]_P )

